programs/whirlpool/src/instructions/two_hop_swap.rs

```diff
@@ -202,6 +202,21 @@ pub fn handler(
         (swap_calc_one, swap_calc_two)
     };
 
+    // All output token should be consumed by the second swap
+    let swap_calc_one_output = if a_to_b_one {
```

Comment:
This is _technically_ a breaking change, though it is still a reasonable guardrail

Reply:
Yep, strictly speaking, it is breaking.
Good news (?) is almost all use of two_hop_swap is from Orca site. (Because its execution number is quite lower than swap).
And Orca backend doesn't use Partial fill now.